mate-panel/libpanel-util/panel-session-manager.c

```diff
@@ -22,18 +22,25 @@
  *	Vincent Untz <vuntz@gnome.org>
  */
 
-#include <dbus/dbus-glib.h>
-
+#include <gio/gio.h>
 #include "panel-cleanup.h"
-#include "panel-dbus-service.h"
-
 #include "panel-session-manager.h"
 
-static GObject *panel_session_manager_constructor (GType                  type,
-						   guint                  n_construct_properties,
-						   GObjectConstructParam *construct_properties);
+struct _PanelSessionManager {
+	GObject     parent;
+	GDBusProxy *proxy;
+};
+
+G_DEFINE_TYPE (PanelSessionManager, panel_session_manager, G_TYPE_OBJECT);
+
+static void
+panel_session_manager_finalize (GObject *object)
```

Comment:
You need to chain-up in finalize... `G_OBJECT_CLASS (...)->finalize (...);`

Reply:
ok, yes, I forgot it.